arex-compare-core/src/main/java/com/arextest/diff/utils/ArexStringEqualsUtil.java

```diff
@@ -50,11 +50,14 @@ public static boolean isEqualsWithoutPrefix(String baseStr, String testStr) {
       }
       return true;
     } else {
-      while (baseIndex < baseValueLen && testIndex < testValueLen) {
+      while (testIndex < testValueLen) {
```

Comment:
这里是否可以把baseValueLen == 0和baseValueLen != 0两种情况合并处理？
```suggestion
      while (testIndex < testValueLen) {
```

Reply:
done